src/coreComponents/unitTests/linearAlgebraTests/testDofManager.cpp

```diff
@@ -1071,6 +1071,39 @@ INSTANTIATE_TYPED_TEST_SUITE_P( Hypre, DofManagerRestrictorTest, HypreInterface,
 INSTANTIATE_TYPED_TEST_SUITE_P( Petsc, DofManagerRestrictorTest, PetscInterface, );
 #endif
 
+TEST( DofManagerRegions, aggregateInitialization )
```

Comment:
I can't object to adding the test as it brings net value. But to be completely honest, moving forward if I have to add such a test for every aggregate I use, I will likely fall back to using `std::tuple` everywhere.

Reply:
While I can understand you and appreciate your honesty, that's concerning.
In this case, `struct` is more meaningful that `std::tuple`.
So using them helps the readability of the code.

While we don't have `designed initializers`, we could add the constructor not to rely in the indexing (like I was writing in https://github.com/GEOSX/GEOSX/pull/1682#discussion_r801851965), or make a test.

Here the test is 8 lines of very dummy code and can save a lot of time to others.
It's really not expensive and it's part of the price to pay to have a code where a lot of scientist can work (even non C++ specialists), or even people who will just pass by for a few days or weeks.

If `GEOSX` collapses under bugs, the product will become less attractive to industrial which will invest less.